vmm/src/cpu.rs

```diff
@@ -487,6 +489,12 @@ impl Vcpu {
             .map_err(Error::VcpuSetGicrBaseAddr)?;
         Ok(())
     }
+
+    #[cfg(feature = "kvm")]
+    pub fn get_kvm_vcpu_raw_fd(&self) -> RawFd {
+        // SAFETY: We hold onto the contract.
+        unsafe { self.vcpu.get_kvm_vcpu_raw_fd() }
+    }
```

Comment:
I would recommend making this an unsafe function as well since the encapsulated logic within this function cannot know that the contract will be respected.